streamx-console/streamx-console-service/src/assembly/script/final.sql

```diff
@@ -572,7 +571,7 @@ UNIQUE KEY `UN_USERNAME` (`NICK_NAME`) USING BTREE
 -- Records of t_user
 -- ----------------------------
 BEGIN;
-INSERT INTO `t_user` VALUES (100000, 'admin', '', 'ats6sdxdqf8vsqjtz0utj461wr', '829b009a6b9cc8ea486a4abbc38e56529f3c6f4c9c6fcd3604b41b1d6eca1a57', 'benjobs@qq.com', '13800000000', '1', NOW(), NULL,NULL,NULL,NULL,NULL );
+INSERT INTO `t_user` VALUES (100000, 'admin', '', 'ats6sdxdqf8vsqjtz0utj461wr', '829b009a6b9cc8ea486a4abbc38e56529f3c6f4c9c6fcd3604b41b1d6eca1a57', 'benjobs@qq.com', '1', NOW(), NULL,NULL,0,NULL,NULL );
```

Comment:
Oh, My God, my private email has been exposed.😫

Reply:
Ok，I will remove set  field `EMAIL` to `NULL`.